R/prep_data.R

```diff
@@ -59,10 +59,18 @@
 #'   into a new category, `other`. If numeric, must be in {0, 1}, and is the
 #'   proportion of observations below which levels will be grouped into other.
 #'   See `recipes::step_other`.
+#' @param PCA Integer or Logical. PCA is done to reduce training time,
+#'   particularly for wide datasets. If integer, represents the number of
```

Comment:
"PCA reduces training time, particularly for wide datasets, though it renders models less interpretable."